W-18136825 Add managed Flex Gateway name (#814)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/_partials/policy-title-headers.adoc

// tag::ui[]
[[connected-mode]]
=== Flex Gateway Connected Mode

> When you apply the policy to your API instance from the UI, the following parameters are displayed:

Reviewer: ```suggestion
When you apply the policy to your API instance from the UI, these parameters are shown:
```

// end::ui[]

// tag::configFile[]
=== Flex Gateway Local Mode

> In Local Mode, you apply the policy to your API via declarative configuration files. Refer to the following policy definition and table of parameters:

Reviewer: ```suggestion
In Local Mode, you apply the policy to your API via declarative configuration files. Refer to these policy definitions and parameters:
```
